src/components/dialogs/criteria-based-filter-dialog-content.js

```diff
@@ -115,14 +184,12 @@ export const CriteriaBasedFilterDialogContent = ({
             if (contentType === ElementType.FILTER) {
                 getFilterById(id)
                     .then((response) => {
-                        setInitialFilter(response);
-                        setEquipmentType(
-                            response.equipmentFilterForm.equipmentType
-                        );
+                        setInitialFilter(backToFrontTweak(response));
+                        let eType = response.equipmentFilterForm.equipmentType;
```

Comment:
etype could be const

Reply:
Yes, it could. But it would trigger the 80 characters per line limit that bother me more.